_test/issue-1330.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net"
+)
+
+type wrappedConn struct {
+	net.Conn
+}
+
+func main() {
+	listener, err := net.Listen("tcp", "127.0.0.1:49153")
+	if err != nil {
+		panic(err)
+	}
+	go func() {
+		_, err := listener.Accept()
+		if err != nil {
+			panic(err)
+		}
+	}()
+
+	dialer := &net.Dialer{
+		LocalAddr: &net.TCPAddr{
+			IP:   net.ParseIP("127.0.0.1"),
+			Port: 0,
+		},
+	}
+
+	conn, err := dialer.Dial("tcp", "127.0.0.1:49153")
+	if err != nil {
+		panic(err)
+	}
+
+	t := &wrappedConn{conn}
+	var w io.Writer = t
+	fmt.Println(w.Write != nil)
```

Comment:
if I add below:

```
	if n, err := w.Write([]byte("hello")); err != nil {
		fmt.Println(err)
	} else {
		fmt.Println(n)
	}
```

I get with yaegi:

```
% yaegi run ./_test/issue-1330.go
true
unexpected fault address 0xc000120978
fatal error: fault
[signal SIGBUS: bus error code=0x2 addr=0xc000120978 pc=0xc000120978]

goroutine 1 [running]:
runtime.throw({0x19eed71, 0x4})
	/Users/mpl/go1/src/runtime/panic.go:1198 +0x71 fp=0xc000492930 sp=0xc000492900 pc=0x103b531
runtime.sigpanic()
	/Users/mpl/go1/src/runtime/signal_unix.go:732 +0x1d6 fp=0xc000492980 sp=0xc000492930 pc=0x1051f16
runtime.call32(0xc0004b7b30, 0xc000118768, 0x2115060, 0x0, 0x0, 0x20, 0xc000492ee8)
	/Users/mpl/go1/src/runtime/asm_amd64.s:626 +0x49 fp=0xc0004929b0 sp=0xc000492980 pc=0x106f669
runtime.reflectcall(0xc0004b7b60, 0x4, 0x4, 0x4bc470, 0xc0, 0x2, 0x2)
	<autogenerated>:1 +0x3c fp=0xc0004929f0 sp=0xc0004929b0 pc=0x107391c
reflect.callMethod(0xc00011b480, 0xc000493180, 0xc000493058, 0xc000493060)
	/Users/mpl/go1/src/reflect/value.go:1055 +0x7d8 fp=0xc000493008 sp=0xc0004929f0 pc=0x10c7638
reflect.callMethod(0xc00011b480, 0xc000493180, 0xc000493058, 0xc000493060)
	<autogenerated>:1 +0x4b fp=0xc000493038 sp=0xc000493008 pc=0x10d200b
reflect.methodValueCall()
	/Users/mpl/go1/src/reflect/asm_amd64.s:80 +0x71 fp=0xc000493180 sp=0xc000493038 pc=0x10d1ed1
github.com/traefik/yaegi/stdlib._io_Writer.Write(...)
	/Users/mpl/src/github.com/traefik/yaegi/stdlib/go1_17_io.go:346
github.com/traefik/yaegi/stdlib.(*_io_Writer).Write(0xc000120cf0, {0xc0004bc428, 0x2, 0x2})
	<autogenerated>:1 +0x3e fp=0xc0004931a8 sp=0xc000493180 pc=0x182d7fe
runtime.call32(0xc0004b7ad0, 0x838e4c8, 0x0, 0x0, 0x0, 0x20, 0xc0004936e8)
	/Users/mpl/go1/src/runtime/asm_amd64.s:626 +0x49 fp=0xc0004931d8 sp=0xc0004931a8 pc=0x106f669
runtime.reflectcall(0x18cf360, 0xc000120c00, 0x4, 0x1a07c16, 0x0, 0x12, 0x18cf360)
	<autogenerated>:1 +0x3c fp=0xc000493218 sp=0xc0004931d8 pc=0x107391c
reflect.Value.call({0x192c6c0, 0xc0004be450, 0xc000493860}, {0x19ed752, 0x4}, {0xc000120d68, 0x1, 0x8371800})
	/Users/mpl/go1/src/reflect/value.go:543 +0x814 fp=0xc000493808 sp=0xc000493218 pc=0x10c4974
reflect.Value.Call({0x192c6c0, 0xc0004be450, 0x10530b2}, {0xc000120d68, 0x1, 0x1})
	/Users/mpl/go1/src/reflect/value.go:339 +0xc5 fp=0xc000493880 sp=0xc000493808 pc=0x10c3f25
github.com/traefik/yaegi/interp.callBin.func1({0x192c6c0, 0xc0004be450, 0x18cf360}, {0xc000120d68, 0xc000493960, 0x13511e5})
	/Users/mpl/src/github.com/traefik/yaegi/interp/run.go:1448 +0x28 fp=0xc0004938c0 sp=0xc000493880 pc=0x139cd68
github.com/traefik/yaegi/interp.callBin.func8(0xc00036a210)
	/Users/mpl/src/github.com/traefik/yaegi/interp/run.go:1589 +0x158 fp=0xc000493970 sp=0xc0004938c0 pc=0x135c378
github.com/traefik/yaegi/interp.runCfg(0xc000496a20, 0xc00036a210, 0xc000373e60, 0xc000493a80)
	/Users/mpl/src/github.com/traefik/yaegi/interp/run.go:201 +0x2ac fp=0xc000493a18 sp=0xc000493970 pc=0x134d42c
github.com/traefik/yaegi/interp.(*Interpreter).run(0xc000368000, 0xc000373e60, 0xc00036a000)
	/Users/mpl/src/github.com/traefik/yaegi/interp/run.go:120 +0x385 fp=0xc000493aa8 sp=0xc000493a18 pc=0x134cac5
github.com/traefik/yaegi/interp.(*Interpreter).Execute(0xc000368000, 0xc0004b67e0)
	/Users/mpl/src/github.com/traefik/yaegi/interp/program.go:157 +0x24b fp=0xc000493b40 sp=0xc000493aa8 pc=0x134bd6b
github.com/traefik/yaegi/interp.(*Interpreter).eval(0xc000368000, {0xc000486840, 0x298}, {0x7ffeefbff6c2, 0xc0004862c0}, 0x98)
	/Users/mpl/src/github.com/traefik/yaegi/interp/interp.go:615 +0x5c fp=0xc000493b80 sp=0xc000493b40 pc=0x130575c
github.com/traefik/yaegi/interp.(*Interpreter).EvalPath(0xc000368000, {0x7ffeefbff6c2, 0x15})
	/Users/mpl/src/github.com/traefik/yaegi/interp/interp.go:508 +0xab fp=0xc000493bc0 sp=0xc000493b80 pc=0x130486b
main.runFile(0x7ffeefbff6c2, {0x7ffeefbff6c2, 0x15}, 0x0)
	/Users/mpl/src/github.com/traefik/yaegi/cmd/yaegi/run.go:153 +0x8f fp=0xc000493c20 sp=0xc000493bc0 pc=0x187d9af
main.run({0xc0000321d0, 0xc000000101, 0xc0000606b8})
	/Users/mpl/src/github.com/traefik/yaegi/cmd/yaegi/run.go:116 +0xbec fp=0xc000493e68 sp=0xc000493c20 pc=0x187d6ac
main.main()
	/Users/mpl/src/github.com/traefik/yaegi/cmd/yaegi/yaegi.go:133 +0xd5 fp=0xc000493f80 sp=0xc000493e68 pc=0x187f775
runtime.main()
	/Users/mpl/go1/src/runtime/proc.go:255 +0x227 fp=0xc000493fe0 sp=0xc000493f80 pc=0x103dc87
runtime.goexit()
	/Users/mpl/go1/src/runtime/asm_amd64.s:1581 +0x1 fp=0xc000493fe8 sp=0xc000493fe0 pc=0x10712c1
```

So I don't think the PR is enough as it is, if the wrappedConn is not usable as a writer anyway, right?

Reply:
fixed by 9feee9f